src/track/serato/tags.cpp

```diff
@@ -114,11 +117,11 @@ double SeratoTags::findTimingOffsetMillis(const QString& filePath) {
 
     double timingOffset = 0;
     if (filePath.toLower().endsWith(".mp3")) {
```

Comment:
We should not come up with a 3rd method for determining the file type. The whole file path is not necessary if we only use need and use the file type for determining the offset.

There are different options. One can be found in trackmetadata_file.h, including a type-safe enum:
```
FileType getFileTypeFromFileName(const QString& fileName);
```

Reply:
Another option would be `QString SoundSource::getFileExtensionFromUrl(QUrl url);` that returns the string that the track uses for the *file type* property. But I guess that the metadata enum fits better here.

Reply:
Done.